### PR TITLE
Fix scrollToBottom && remove scrollsToBottomOnFirstLayout

### PR DESCRIPTION
**Pull request by zhongwuzw**

### CODE REVIEW

File: Sources/Views/MessagesCollectionView.swift [hunk 1]
```diff
@@ -65,7 +65,20 @@ open class MessagesCollectionView: UICollectionView {
 
     public func scrollToBottom(animated: Bool = false) {
         guard let indexPath = indexPathForLastItem else { return }
-        scrollToItem(at: indexPath, at: .bottom, animated: animated)
+        
+        let collectionViewContentHeight = collectionViewLayout.collectionViewContentSize.height
+        let isContentTooSmall = (collectionViewContentHeight < bounds.height * 2)
```
Comment (SD10) [inline Sources/Views/MessagesCollectionView.swift:70]: Why `bounds.height * 2` here?

**Comment by SD10** [comment]
@zhongwuzw This looks good. I fixed the problem with the `MessageLabel` font. 

The only thing I notice about this change is there is a small flicker from using batch updates 🤔

**Comment by SD10** [comment]
@zhongwuzw I also want to mention you don't have to work from a fork of the project, you're a contributor so you can work from branches on the main MessageKit repo 😅

**Comment by zhongwuzw** [comment]
@SD10 
> The only thing I notice about this change is there is a small flicker from using batch updates
I think we can use `scrollToBottom(animated: true)`, the flicker is because of not animated.

File: Sources/Views/MessagesCollectionView.swift (hunk 1 above)
Comment (zhongwuzw) [reply Sources/Views/MessagesCollectionView.swift:70]: The reason is `scrollToItem` would sometimes invalid when `collectionViewContentSize`'s `height` slightly larger than `collectionView`'s `height`.

Comment (SD10) [reply Sources/Views/MessagesCollectionView.swift:70]: Yes, I understand that, but why did we decide to use `* 2`? Is this just a random number?

Comment (zhongwuzw) [reply Sources/Views/MessagesCollectionView.swift:70]: Nope, it's based on my tests, I found `scrollToItem` sometimes invalid when last cell's maxY greater than bounds's height but less than 2 * bounds's height.

**Review by SD10 — CHANGES_REQUESTED** [review]
@zhongwuzw Can you add a CHANGELOG entry under **Removed** for the `scrollsToBottomOnFirstLayout` flag? And another under **Fixed** for `scrollToBottom()`?

**Comment by zhongwuzw** [comment]
@SD10 , I added the changelog.

**Review by SD10 — APPROVED** [review]
Awesome work @zhongwuzw 💯

**Pull request merged**